components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -274,8 +274,6 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
             categoryMenu = (
                 <SidebarCategoryMenu
                     category={category}
-                    isMenuOpen={this.state.isMenuOpen}
-                    onToggleMenu={this.handleMenuToggle}
```

Comment:
I noticed the styling for this component changes when the menu is open. Is that something we'll need to worry about with this? It's possible that extra CSS class that's applied is for dealing with overflow though which we shouldn't have to do anymore